cmd/cni-metrics-helper/main.go

```diff
@@ -80,6 +81,16 @@ func main() {
 		}
 	}
 
+	pullIntervalEnv, found := os.LookupEnv("PULL_INTERVAL")
```

Comment:
nit: Can we use a more descriptive name, maybe `METRIC_PULL_INTERVAL` or `METRIC_PULL_INTERVAL_SECS`?

Reply:
I am more than happy to switch the ENV VAR name. I used this name to preserve the golang var name as much as possible. I'll switch it now.